Ladybird/BrowserWindow.cpp

```diff
@@ -504,3 +506,19 @@ void BrowserWindow::copy_selected_text()
         clipboard->setText(qstring_from_ak_deprecated_string(text));
     }
 }
+
+bool BrowserWindow::eventFilter(QObject* obj, QEvent* event)
+{
+    if (event->type() == QEvent::MouseButtonRelease) {
```

Comment:
is it possible to middle click outside the tab and release the button on it to close it? 🤔

itʼd be good to have proper clicks only, i.e. pressing middle button on the tab and releasing it there…

Reply:
It works only if you press and release staying on the tab :)

Reply:
okay; was not sure! 😅